edi_oca/models/edi_backend.py

```diff
@@ -579,6 +577,17 @@ def _exchange_receive(self, exchange_record):
             return component.receive()
         raise NotImplementedError()
 
+    def _process_received_content(self, exchange_record, content):
+        if not content:
```

Comment:
we have 2 cases:
- content is None: -> because of FileNotFoundError/OSError, we don't do anything
- `len(content) == 0`: file is empty

Reply:
in which case do we have False?

Reply:
For now, we don't have any case with `False` value for content, but in the future, maybe it will support some storage that wants to ignore the error and return `False` value

I think it will be great if we bypass both `None` and `False` in case the storage wants to ignore the error by returning one of them

Reply:
I don't really care but usually we try to cover existing cases not future potential cases